packages/core/src/components/Tipseen/__stories__/Tipseen.mdx

```diff
@@ -96,6 +96,7 @@ In this case, the Tipseen position will be the right corner of the screen and wi
               modifiers={modifiers}
               position={Tipseen.positions.RIGHT}
               color={Tipseen.colors.INVERTED}
+              showDelay={100}
```

Comment:
Didn't we decide to make it the default?

Reply:
That was the hideDelay. The showDelay I added to the tipseens just for the storybook so they don't disappear after hover.

Reply:
I thought the showDelay was added for that, is there any way to avoid it? it would be best to show a minimal code as possible for each story

Reply:
Made it default